.github/workflows/ci.yml

```diff
@@ -89,6 +89,39 @@ jobs:
       - name: "Test"
         run: nix develop -L --no-update-lock-file --command just test-nix-plugins
 
+  nef-dev:
+    name: "Nix expression tests"
+    runs-on: "ubuntu-22.04-8core" 
```

Comment:
question nonblocking: are you thinking one system since they're unit tests?

Reply:
Yes as long as they don't build system specific stuff I don't think we need to run them on the whole matrix of systems, and build tests seem to end up in the cli parts.